### ISSUE TITLE
Document IAM permissions required for s3 sync target

### ISSUE THREAD

**Issue by DanielHeath** [camlistore/camlistore#799]
I've created a camlistore user for my AWS account - I'd like to lock it down so that e.g. keys getting leaked don't let someone delete my backups.

**Comment by DanielHeath** [comment]
I'm going to try using the following policy document - from the code it looks like it'll be enough.
```
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Resource": [
                "arn:aws:s3:::YOUR_BUCKET_NAME/*"
            ],
            "Sid": "Stmt1464826210000",
            "Effect": "Allow",
            "Action": [
                "s3:DeleteObject",
                "s3:GetBucketLocation",
                "s3:GetObject",
                "s3:ListBucket",
                "s3:PutObject"
            ]
        }
    ]
}
```

**Comment by edrex** [comment]
Previous discussion: https://groups.google.com/forum/#!searchin/camlistore/s3$20policy/camlistore/W8GE1fpxtbI/mgbFa7GfBL0J

This could be documented in the /pkg/blobserver/s3 source, but that doesn't seem like a good fit.

Let's add a new page, /doc/storage.md, with sections with operational guidance for the various storage implementations. We could make one page per implementation, but I think they would be pretty tiny, at least initially (can always split them later). This page should be linked from /doc/server-config.md

@DanielHeath, do you want to take this?

**Comment by DanielHeath** [comment]
I've put together a terse overview - see //camlistore-review.googlesource.com/6737

**Comment by willnorris** [comment]
fixed in 0b682c67112d87abbe16018b39627524a31251c0
